docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,125 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configuring high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+
+Grafana Mimir can deduplicate data from high-availability (HA) Prometheus setup. In this guide, you will see how to configure
+the deduplication for Grafana Mimir helm deployment using external Consul.
+
+## Before you begin
+
+You need to have a Grafana Mimir installed.
+
+Refer to [Configuring High Availability]({{< relref "../configuring-high-availability-deduplication.md" >}}) documents
+for high level description on the concept how Mimir deduplicate incoming HA samples.
+
+You also have to configure HA for Prometheus or Grafana Agent. Last, you need a KV store. In this guide you will
+use Consul. You will be guided on the setup if you haven't had one.
+
+## Configure Prometheus or Grafana Agent to send HA external labels
+
+Configure Prometheus or Grafana Agent HA setup by setting label called `cluster` and `__replica__`. These two labels
+name are default labels for HA setup in Grafana Mimir. If you want to change the labels, make sure to change it in Mimir
+too to make Grafana Mimir and Prometheus/Grafana Agent config matches with each other, otherwise HA deduplication
+wouldn't work. `cluster` label value must be same across replica belong to the same cluster. `__replica__` must be
+unique across different replica in the cluster.
+
+```yaml
+global:
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  evaluation_interval: 1m
+  external_labels:
+    __replica__: replica-1
+    cluster: my-prometheus
+```
+
+Reload or restart Prometheus or Grafana Agent after updating the configuration.
+
+## Install Consul using Helm
+
+Install Consul in Kubernetes using
+[Consul helm chart](https://github.com/hashicorp/consul-k8s/tree/main/charts/consul). Follow the documentation to get
+the chart and installing a Consul release. Put a note on the Consul endpoint, because you will need it for Mimir
+configuration.
+
+## Setup Mimir HA config
+
+You can set Mimir HA deduplication configuration in global level or tenant level.
+
+### HA Deduplication Global
+
+You need a Mimir setup installed by Helm. Add the following configuration to your `values.yaml` to configure HA
```

Comment:
nitpick about values.yaml. I think we've used custom.yaml in a few articles so far to refer to the values file that the user supplies. It makes it obvious that it's a different file from the values.yaml in the root of the helm chart. Those values.yaml are the default ones and helm uses them every time, even without providing a flag.